docs/source/get_started/minimal_kedro_project.md

```diff
@@ -0,0 +1,176 @@
+# Create a Minimal Kedro Project
+This documentation aims to explain the essential components of a minimal Kedro project. The guide begins with a blank project and gradually introduces the necessary elements. While most users typically start with a [project template]((./new_project.md)) or adapt an existing Python project, this guide will help you understand the core concepts and how to customise them to suit your specific needs.
```

Comment:
https://github.com/kedro-org/kedro/pull/4305#discussion_r1836815371

I think the idea is that, starting with a blank project isn't something that most people would do, so the goal of this documentation focus on explains the concept, but not a step-by-step guide that one should follow in reality.

Maybe I should flip the sentence like this?

While most users typically start with a [project template](./new_project.md)) or adapt an existing Python project, this guide begins with a blank project and gradually introduces the necessary elements. This will help you understand the core concepts and how to customise them to suit your specific needs.